usr/src/uts/common/fs/zfs/lua/lstrlib.c

```diff
@@ -34,7 +34,6 @@
  */
 #define tolower(C)	(((C) >= 'A' && (C) <= 'Z') ? (C) - 'A' + 'a' : (C))
 #define toupper(C)      (((C) >= 'a' && (C) <= 'z') ? (C) - 'a' + 'A': (C))
-#define iscntrl(C)	((((C) >= 0) && ((C) <= 0x1f)) || ((C) == 0x7f))
```

Comment:
There's no need to turn this into function, simply remove the ```((C) >= 0)``` part - the compiler whines about checking if ```uchar``` is greater or equal to 0, and it's always true (and that check is annoying).

P.S. it's sad to see yet another bunch of is* stuff added to kernel, they really need to be unified.

Reply:
@yuripv Are you satisfied with this review?

Igor, I generally lean toward "the less change the better", so I'd probably go as Yuri suggests and just remove the test for zero.

Reply:
@gwr I'm not sure, because it can be done the same for proposed mdb version too?
why not to reuse tested as long time realization?

Reply:
I don't have strong feelings either way.  I just wanted to confirm that the discussion was concluded to the satisfaction of your reviewers.

Reply:
It does seem simpler to just remove the `C >= 0` from the macro, and more consistent with the other `is*()` in this .c file being macros rather than functions.  But I'm fine either way.